src/nvim/highlight_group.c

```diff
@@ -1045,6 +1029,11 @@ void do_highlight(const char *line, const bool forceit, const bool init)
     }
   }
 
+  const char *key_start;
+  const char *arg_start;
```

Comment:
Scope of these two can be reduced even more (cppcheck is nice ;) )

Reply:
Just got it set up in null-ls. This is amazing.